src/player.js

```diff
@@ -558,8 +558,8 @@ export default class Player extends FakeEventTarget {
       this.dispatchEvent(new FakeEvent(CustomEventType.PLAYBACK_START));
       if (!this.src) {
         this._prepareVideoElement();
+        this.load();
```

Comment:
maybe it is better to put the check inside load function? as this can also be called from external api